src/containers/VerletListHelpers.h

```diff
@@ -78,20 +79,20 @@ class VerletListHelpers {
       }
     }
 
-    virtual void SoAFunctor(SoA &soa1, SoA &soa2, bool /*newton3*/ = true) override {
+    virtual void SoAFunctor(SoA<Particle> &soa1, SoA<Particle> &soa2, bool /*newton3*/ = true) override {
```

Comment:
AutoPas always has only one Particle class, that has to be specified from the outside.
However, that class can contain multiple types. How exactly this will be handled, no idea yet.